__tests__/supertest.test.js

```diff
@@ -0,0 +1,19 @@
+const request = require("supertest");
+const app = require("../app");
+const connection = require("../db/connection");
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => {
+    return seed(testData);
+})
+afterAll(() => connection.end());
+
+describe("GET /api/categories", () => {
+  it("serves an array of all categories", () => {
+    return request(app).get("/api/categories").expect(200).then((res) => {
+        expect(res.body.categories.length).toBe(4);
```

Comment:
you need to test more than just the length of the array here, you should be checking that each object contains the keys you expect using .toMatchObject or .objectContaining